### PR TITLE
Updates sim to use proper CAD properties

### PR DESCRIPTION
**Pull request by redPlover**

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/scoring/ScoringSubsystem.java [hunk 1]
```diff
@@ -284,7 +290,10 @@ public void periodic() {
                                 FieldLocations.RED_STAGE))) {
             aimerIo.setAngleClampsRad(0, 0);
         } else {
-            aimerIo.setAngleClampsRad(0, ScoringConstants.aimMaxAngleRadians);
+            double elevatorLimit =
+                    (Math.PI / 2 / 0.45) * (elevatorPositionSupplier.get() - 0.45) + Math.PI / 2;
```
Comment (jkleiber) [inline src/main/java/frc/robot/subsystems/scoring/ScoringSubsystem.java:294]: What does this equation do / can it be documented better to explain what each of the magic numbers mean (pi / 2, 0.45)

**Review by jkleiber — COMMENTED** [review]
A couple minor comments

File: src/main/java/frc/robot/subsystems/scoring/AimerIOSim.java [hunk 2]
```diff
@@ -33,7 +32,9 @@ public class AimerIOSim implements AimerIO {
                     ScoringConstants.aimerkV,
                     ScoringConstants.aimerkA);
     private final TrapezoidProfile profile =
-            new TrapezoidProfile(new TrapezoidProfile.Constraints(0.8, 0.5));
+            new TrapezoidProfile(
+                    new TrapezoidProfile.Constraints(
+                            ScoringConstants.aimCruiseVelocity, ScoringConstants.aimAcceleration));
```
[the remark this answers is not in the document]
Comment (redPlover) [reply src/main/java/frc/robot/subsystems/scoring/AimerIOSim.java:37]: There are supposed to be 10. This changed based off the new sim constants.

File: src/main/java/frc/robot/subsystems/scoring/ScoringSubsystem.java [hunk 3]
```diff
@@ -284,7 +297,17 @@ public void periodic() {
                                 FieldLocations.RED_STAGE))) {
             aimerIo.setAngleClampsRad(0, 0);
         } else {
-            aimerIo.setAngleClampsRad(0, ScoringConstants.aimMaxAngleRadians);
+            // Linear equation in point-slope form to calculate the arm's limit based on the
+            // elevator position
+            double maxElevatorPosition = 0.45; // x-intercept
+            double maxAimAngle = Math.PI / 2; // y-intercept
```
Comment (jkleiber) [inline src/main/java/frc/robot/subsystems/scoring/ScoringSubsystem.java:303]: Can we add these to Constants.java

**Review by jkleiber — APPROVED** [review]

**Pull request merged**